packages/components/data-table-manager/src/data-table-manager.spec.js

```diff
@@ -41,8 +45,37 @@ const TestComponent = (props) => {
     </DataTableManager>
   );
 };
-/* eslint-enable react/prop-types */
 
+const DecoupledDatatableTestComponent = (props) => {
```

Comment:
I don't see we're actually testing the decoupling here as it seems the provider state is not used in the `TestTable`.

`DataTableManager` and `TestTable` components should be in different leafs of the component's tree.

Reply:
Updated here: https://github.com/commercetools/ui-kit/pull/2807/commits/bd8447871f0d99ac366b5adc9d865055c9a4506e

Thank you